packages/slice-machine/src/features/customTypes/customTypesBuilder/GenerateSliceWithAiModal/SliceCard.tsx

```diff
@@ -9,12 +9,19 @@ interface SliceCardProps {
 export function SliceCard(props: SliceCardProps) {
   const { slice } = props;
 
-  const loading = slice.status === "uploading";
-  const error = slice.status === "uploadError";
+  const loading = ["uploading", "generating"].includes(slice.status);
```

Comment:
while this is more elegant than checking each status individually in an or as you do below, the latter is safer because typescript can help you avoid mistakes; array.includes works with any string

you can try this here, you can do `["mistake", "uploading"].includes(slice.status)` and it won't complain, but you can't do `slice.status === "mistake" || slice.status === "uploading"` because typescript won't allow it

Reply:
Good point. Thanks, I fixed it ☑️